jme3-examples/src/main/java/jme3test/awt/TestCanvas.java

```diff
@@ -234,7 +238,7 @@ public static void createCanvas(String appClass){
     }
 
     public static void startApp(){
-        app.startCanvas();
+        /*app.startCanvas();*/

```

Comment:
Code in comments are bad. Also this test should still work on both LWJGL 2 and LWJGL 3. I think it is important that LwjglCanvas works the same. It should handle all disparities internally and not require any shenanigans from the user.

Reply:
Correction in pending commits....